File: tasks/implementations/node/tasks/file_io_large.js
const fs = require('fs');
module.exports = function(size, fixturesRoot) {
  const path = `${fixturesRoot}/generated/file_io/${size}.txt`;
  let total = 0;
  for (const line of fs.readFileSync(path, 'utf8').trim().split(/\s+/)) {
    total += Number(line);
  }
  console.log(total);
};

File: tasks/implementations/java/tasks/file_io_large.java
import java.nio.file.*;

class FileIOLarge {
    public static void run(String size, String fixturesRoot) throws Exception {
        long total = 0;
        for (String line : Files.readAllLines(Path.of(fixturesRoot, "generated", "file_io", size + ".txt"))) {
            total += Integer.parseInt(line.trim());
        }
        System.out.println(total);
    }
}
